### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by dil-bpecor** [announcekitapp/announcekit-react#20]
Following this documentation for the "What's New" Link:
https://announcekit.app/docs/reactjs

Using Wave Accessibility tool shows link as empty which also causes a keyboard navigation issue as you cannot get to the "what New" by tabbing through the page to go to it you have to use a mouse.
![image](https://user-images.githubusercontent.com/46458819/131760047-0ff9ba4a-111d-4f01-8e32-c73639bfa50f.png)
